src/game/client.cpp

```diff
@@ -1482,6 +1482,142 @@ namespace client
     }
     //ICOMMAND(0, sendmap, "", (), sendmap());
 
+    static void startcmd(char *map, char *mode_str, char *muts_str)
```

Comment:
You could likely use `std::string` instead of these char arrays.

Reply:
I could but that doesn't give me any benefit. The only change it would make is that I would have to type ".c_str()" more inside the function.